### PR TITLE
[zh] translate docs: components.md

### PR DESCRIPTION
**Pull request by chaosi-zju**
Add one of the following kinds:

/kind documentation

thank you for your review @Poor12 @windsonsea @RainbowMango

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/components.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+title: 关键组件
+---
+本文档概述了一个功能完整且正常工作的 Karmada 所需的组件。
+
+![components](../resources/general/components.png)
+
+## Control Plane Components
+
+一个完整且可工作的 Karmada 控制平面由以下部件组成。karmada-agent 可能是可选的，这取决于
+[集群注册模式](../userguide/clustermanager/cluster-registration)。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/components.md:11]:
```suggestion
一个完整且可工作的 Karmada 控制平面由以下组件组成。karmada-agent 可以是可选的，
这取决于[集群注册模式](../userguide/clustermanager/cluster-registration)。
```
不建议在汉字中间换行，可以在中英、数字、标点处换行

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/components.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+一致且高可用的键值存储，用作 Karmada 的所有 Karmada/Kubernetes 资源对象数据的后台数据库。
+
+如果你的 Karmada 使用 etcd 作为其后台数据库， 请确保你针对这些数据有一份 备份计划。
+
+你可以在官方[文档](https://etcd.io/docs/)中找到有关 etcd 的深入知识。
+
+### karmada-agent
+
+Karmada 有 Push 和 Pull 两种 [集群注册模式](../userguide/clustermanager/cluster-registration)， karmada-agent 应部署在每个 Pull 模式的成员集群上。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/components.md:71]:
```suggestion
Karmada 有 Push 和 Pull 两种[集群注册模式](../userguide/clustermanager/cluster-registration)，karmada-agent 应部署在每个 Pull 模式的成员集群上。
```
以渲染/预览的结果来确定是否要加空格，下同

**Comment by chaosi-zju** [comment]
Do you have any further comments? If you have any other suggestions, please let me know. @Poor12 @windsonsea 

If no more comments, may this PR be merged ? @RainbowMango

**Comment by windsonsea** [comment]
Overall LGTM.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**